src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -55875,17 +56233,18 @@ void M0<T>(T x0, T z0)
             M2(y0) = z0;
             x0.ToString();
             y0.ToString();
+            M2(x0).ToString();
```

Comment:
>M2(x0).ToString(); [](start = 12, length = 18)

The ```x0.ToString()``` line above actually changes the flow sate of ```x0```, consider placing this line above it, so that both ```M2(x0)```` calls are based on the same state.